### PR TITLE
Removing todo

### PR DESCRIPTION
**Pull request by vinagesh**
## Checklist
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [ ] This pull-request is submitted against the `master` branch.

### CODE REVIEW

File: iothub/device/src/DeviceAuthenticationWithX509Certificate.cs [hunk 1]
```diff
@@ -45,7 +45,6 @@ public string DeviceId
         /// <summary>
         /// Full chain of certificates from the one used to sign the device certificate to the one uploaded to the service. Private keys are not required for these certificates.
         /// This is only supported on AMQP_Tcp_Only and Mqtt_Tcp_Only
-        /// See <see href="TODO (Sample ref) - Sindhu"/>
```
Comment (vinagesh) [inline iothub/device/src/DeviceAuthenticationWithX509Certificate.cs:48]: Realized samples depend on packages and not project ref. So I cannot send out a PR in samples repo till we release a new package.

**Review by azabbasi — APPROVED** [review]

**Review by abhipsaMisra — APPROVED** [review]

**Comment by abhipsaMisra** [comment]
One more: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/device/src/Common/Security/CertificateInstaller.cs#L23 :)

**Pull request merged**